chargeOclock/Source/Models/Client.swift

```diff
@@ -14,8 +14,8 @@ protocol ClientSettable {
 
 class Client {
 
-	var id: Int = 0
-	var isDirty: Bool = false
+	var id: Int? = nil
+	var isDirty: Bool!
```

Comment:
Ouch, that's right, I've overlooked something crucial. I made a terrible mistake. This is not what I intended. My intention was to leave it uninitialised, so just `var isDirty: Bool`, without `!`. That's the scenario that I was talking about. Now it forces you to initialise it in every initialiser that you implement. 
What do you think about it now? Should `isDirty` be left uninitialised, or you prefer better the giving it the default value?